capa/v28.1.1/release.yaml

```diff
@@ -0,0 +1,128 @@
+# Generated with:
+# devctl release create --name 28.1.1 --provider capa --base 28.1.0 --app security-bundle@1.7.2
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: aws-28.1.1
+spec:
+  apps:
+  - dependsOn:
+    - cloud-provider-aws
+    name: aws-ebs-csi-driver
+    version: 2.30.1
+  - dependsOn:
+    - cert-manager
+    name: aws-ebs-csi-driver-servicemonitors
+    version: 0.1.0
+  - dependsOn:
+    - cert-manager
+    name: aws-pod-identity-webhook
+    version: 1.16.0
+  - name: capi-node-labeler
+    version: 0.5.0
+  - dependsOn:
+    - kyverno
+    name: cert-exporter
+    version: 2.9.0
+  - dependsOn:
+    - prometheus-operator-crd
+    name: cert-manager
+    version: 3.7.9
+  - dependsOn:
+    - prometheus-operator-crd
+    name: chart-operator-extensions
+    version: 1.1.2
+  - name: cilium
+    version: 0.25.1
+  - name: cilium-crossplane-resources
+    version: 0.1.0
+  - dependsOn:
+    - prometheus-operator-crd
+    name: cilium-servicemonitors
+    version: 0.1.2
+  - dependsOn:
+    - vertical-pod-autoscaler-crd
+    name: cloud-provider-aws
+    version: 1.28.6-gs1
+  - dependsOn:
+    - kyverno
+    name: cluster-autoscaler
+    version: 1.28.5-gs1
+  - dependsOn:
+    - cilium
+    name: coredns
+    version: 1.21.0
+  - dependsOn:
+    - kyverno
+    name: etcd-k8s-res-count-exporter
+    version: 1.10.0
+  - dependsOn:
+    - prometheus-operator-crd
+    name: external-dns
+    version: 3.1.0
+  - dependsOn:
+    - cert-manager
+    name: irsa-servicemonitors
+    version: 0.0.1
+  - dependsOn:
+    - kyverno
+    name: k8s-audit-metrics
+    version: 0.9.0
+  - dependsOn:
+    - kyverno
+    name: k8s-dns-node-cache
+    version: 2.6.2
+  - dependsOn:
+    - kyverno
+    name: metrics-server
+    version: 2.4.2
+  - dependsOn:
+    - prometheus-operator-crd
+    name: net-exporter
+    version: 1.19.0
+  - catalog: cluster
+    dependsOn:
+    - cilium
+    name: network-policies
+    version: 0.1.1
+  - dependsOn:
+    - kyverno
+    name: node-exporter
+    version: 1.19.0
+  - dependsOn:
+    - coredns
+    name: observability-bundle
+    version: 1.3.4
```

Comment:
```suggestion
    version: 1.6.1
```

Reply:
This is a pure patch release for v28.1.0.